integration/inspektor-gadget/integration_test.go

```diff
@@ -1068,41 +1069,63 @@ func TestNetworkGraph(t *testing.T) {
 	}
```

Comment:
I'd suggest to rebase on top of https://github.com/inspektor-gadget/inspektor-gadget/pull/1251 given that it's going to be merged. (Sorry for the conflicts, but it happens when we're working on many things in parallel).

Reply:
Done.